SUMMARY.md

```diff
@@ -41,6 +40,52 @@
    * [Waterline](databases/waterline.md)
    * [LevelUP](databases/levelup.md)
    * [RethinkDB](databases/rethinkdb.md)
+* [Ecosystem & APIs](/ecosystem/readme.md)
+   * [authentication-popups](/ecosystem/authentication-popups/readme.md)
```

Comment:
I guess they can be single files.  If there's one that needs more than one page, we can split it into a folder.